src/views/TableDetail.vue

```diff
@@ -85,14 +85,32 @@
           height="calc(100vh - 123px)"
           class="table-details"
           :headers="dataTableHeaders"
+          hide-default-header
           :items="dataTableRows"
           :footer-props="{
             itemsPerPageOptions: [10, 20, 50, 100],
             showFirstLastPage: true,
           }"
           :server-items-length="tableSize"
           :options.sync="pagination"
-        />
+        >
+          <template v-slot:header>
+            <thead dark>
+              <tr>
+                <th
+                  v-for="(header, i) in dataTableHeaders"
+                  :key="i"
+                  class="pt-2 pb-4"
```

Comment:
It looks a little strange to me that the headers are not centered on the page. I'll defer to Jared though, for final say on this.

```suggestion
                  class="pt-2 pb-2"
```

Reply:
@jtomeck, can you weigh in on this? I was copying this from a [different place in the code with custom headers](https://github.com/multinet-app/multinet-client/blob/master/src/components/TableDialog.vue#L133), but @JackWilb is right, this throws off vertical alignment. With Jack's suggestion, it looks better.

Reply:
I can see you closed this in 865ef9f and Jared approved. I'll mark as resolved